pkg/kubebench/kubebench.go

```diff
@@ -97,8 +134,9 @@ func deployJob(ctx context.Context, clientset *kubernetes.Clientset, params *par
 	if err := decoder.Decode(job); err != nil {
 		return "", err
 	}
+	kubebenchImg := params.Registry + "/kube-bench:" + params.KubebenchTag
```

Comment:
This. means we require the package name as `kube-bench`, but the tag is configurable, correct?

Reply:
yes